src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -564,16 +528,19 @@ Namespace Microsoft.CodeAnalysis.Operations
                 Return Create(boundConversion.Operand)
             End If
 
-            Dim type As ITypeSymbol = boundConversion.Type
-            Dim constantValue As [Optional](Of Object) = ConvertToOptional(boundConversion.ConstantValueOpt)
-            Dim isImplicit As Boolean = boundConversion.WasCompilerGenerated OrElse Not boundConversion.ExplicitCastInCode
-
-            Dim conversionOperandAndMethod = GetConversionInfo(boundConversion)
-            Dim boundOperand As BoundExpression = conversionOperandAndMethod.Operand
-            Dim conversion As Conversion = CreateConversion(boundConversion)
+            Return CreateBoundConversionOrCastOperation(boundConversion, isTryCast:=False)
+        End Function
 
-            If boundOperand.Syntax Is boundConversion.Syntax Then
-                If boundOperand.Kind = BoundKind.ConvertedTupleLiteral AndAlso boundOperand.Type = boundConversion.Type Then
+        Private Function CreateBoundConversionOrCastOperation(boundConversionOrCast As BoundConversionOrCast, isTryCast As Boolean) As IOperation
+            Dim isChecked As Boolean = False
+            Dim type As ITypeSymbol = boundConversionOrCast.Type
+            Dim constantValue As [Optional](Of Object) = ConvertToOptional(boundConversionOrCast.ConstantValueOpt)
+            Dim syntax As SyntaxNode = boundConversionOrCast.Syntax
+            Dim isImplicit As Boolean = boundConversionOrCast.WasCompilerGenerated OrElse Not boundConversionOrCast.ExplicitCastInCode
+
+            Dim boundOperand = GetConversionOperand(boundConversionOrCast)
```

Comment:
>Dim boundOperand [](start = 12, length = 16)

Please spell out the type. Type can be omitted only if it is spelled out in the initializer.